app/lib/shared/markdown.dart

```diff
@@ -81,7 +98,24 @@ String _renderSafeHtml(List<m.Node> nodes) {
       return _whitelistedClassNames.contains(cn);
     },
   );
-  return html + '\n';
+  return inlineOnly
+      ? html.replaceAll(_multiLineBreakRegExp, '<br />\n')
```

Comment:
Can we please do this before the call to `_keepOnlyInlineElements` and then explicitly allow `<br>`... that way we don't ever output HTML that didn't go through some variant of `sanitizeHtml`.

My point: if hacking HTML manually please do so before any calls to `sanitizeHtml`.